test/test_images/performance/main.go

```diff
@@ -31,6 +32,8 @@ import (
 )
 
 // flags for the image
+type Tags []string
```

Comment:
You could keep the type and its methods together in the code since they are related.

Reply:
where exactly?

Reply:
I think @antoineco means together in this file. I.e. move the definition of the type down to immediately above where its methods are defined at https://github.com/knative/eventing/pull/1943/files#diff-669e0a9c7dc91067c70cf3027efcbe49R55.